irohad/validation/stateless_validator.cpp

```diff
@@ -45,11 +46,11 @@ namespace iroha {
 
       // tx is not sent from future
       {
-        if (now.count() < transaction.created_ts) {
+        if (now.count() < transaction.created_ts + 300 && transaction.created_ts < now.count() - 1440*60000) {
```

Comment:
Yes, I put in those magic numbers as only a very temporary example. I am sure Mizuki can make it better.